webapp/src/main/webapp/WEB-INF/view/scripts/jquery-ui/js/openmrsSearch.js

```diff
@@ -709,11 +754,12 @@ function OpenmrsSearch(div, showIncludeVoided, searchHandler, selectionHandler,
                     if(matchCount == self._results.length){
                         spinnerObj.css("visibility", "hidden");
                         loadingMsgObj.html("");
-                        if(self._results.length % self._table.fnSettings()._iDisplayLength == 0)
+                        if(self._results.length % self._table.fnSettings()._iDisplayLength == 0){
                             self._table.numberOfPages = self._results.length/self._table.fnSettings()._iDisplayLength;
-                        else
+						}
+                        else{
                             self._table.numberOfPages = Math.floor(self._results.length/self._table.fnSettings()._iDisplayLength)+1;
-
```

Comment:
@pl9870 i think the blank line here should be removed

Reply:
Will do